python/ray/experimental/dag/py_obj_scanner.py

```diff
@@ -45,7 +45,7 @@ def __init__(self):
         # Register pickler override for DAGNode types.
         from ray.experimental.dag.function_node import FunctionNode
         from ray.experimental.dag.class_node import ClassNode, ClassMethodNode
-        from ray.experimental.dag.input_node import InputNode, InputAtrributeNode
+        from ray.experimental.dag.input_node import InputNode, InputAttributeNode
         from ray.serve.pipeline.deployment_node import DeploymentNode
         from ray.serve.pipeline.deployment_method_node import DeploymentMethodNode
         from ray.serve.pipeline.deployment_function_node import DeploymentFunctionNode
```

Comment:
cc: @ericl we're seeing some CI failures due to importing serve modules here implies ray core user requires serve dependency such as fastapi. Do you have anything in mind that we can remove these local imports and serve import for pickling ?

Reply:
To add a little bit on this: CI won't complain much here as we did a local import.  But during usage, this might be bothering to users, e.g. 
```
@ray.remote
def a(*args, **kwargs):
    pass

a.bind().get_children()
```
will throw errors like `ModuleNotFoundError: No module named 'fastapi'. You can run `pip install "ray[serve]"` to install all Ray Serve dependencies.`